core/templates/dev/head/css/oppia.css

```diff
@@ -135,6 +185,7 @@ md-input-group.md-default-theme label {
 
 /* Attributes for material-icons. */
 .material-icons {
+  line-height: 1;
```

Comment:
Missing unit here?

Reply:
Below as well.

Reply:
Done

Reply:
I checked this. `line-height` attribute with and without the unit have different meanings. Please recheck the use case here (and at places where you've changed this) and then chose the correct one accordingly. (Reference: https://www.w3schools.com/cssref/pr_dim_line-height.asp)

Reply:
Hi @apb7,
yes, line-height: 2 is equal to line-height: 2em, but when these class are inherited the ratio is not maintained in line-height with units as mentioned [here](https://stackoverflow.com/questions/15132671/ems-for-line-height) I think we should go the number ones (with unit)

Reply:
Hmm, I am a bit unsure about adding units to `line-height` attributes since MDN recommends [here](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#Values) that:
> In most cases, this (<number> (unitless)) is the preferred way to set line-height and avoid unexpected results due to inheritance.

I suggest you go for the <number> (unitless) then since:
a. It's prevalent in the codebase.
b. It seems to be the preferred way.

Thanks!

Reply:
Hi @apb7, I've reverted the changes.
The units we were using before are fine and the line-heights added by me are referenced from bower- material library in which units are added based on use case Thanks!